modules/nf-core/pypgx/createinputvcf/tests/main.nf.test

```diff
@@ -0,0 +1,71 @@
+// nf-core modules test pypgx/createinputvcf
+nextflow_process {
+
+    name "Test Process PYPGX_CREATEINPUTVCF"
+    script "../main.nf"
+    process "PYPGX_CREATEINPUTVCF"
+    config "./nextflow.config"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "pypgx"
+    tag "pypgx/createinputvcf"
+
+
+    test("human paired end bam - genome.fa") {
+        when {
+            process {
+                """
+                input[0] = [
+                    [id: 'test'],
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/bam/test.paired_end.markduplicates.sorted.bam', checkIfExists: true),
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/bam/test.paired_end.markduplicates.sorted.bam.bai', checkIfExists: true)
+                ]
+                input[1] = [
+                    [id: 'GRCh38'],
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/genome.fasta', checkIfExists: true)
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    file(process.out.vcf[0][1]).name,
```

Comment:
@nvnieuwk Thank you for the suggestion! I realise I will need additional test data to better test this module. I have tested it before with my own data, but with the current nf-core/test-datasets samples I can only produce an emtpy VCF file (except for the header). 

I already have an ongoing topic on slack on this subject to do this properly, but in the meantime I opened a PR on https://github.com/nf-core/test-datasets/pull/1338 to include some temporary test data in delete_me. I will need that data for different modules of the same tool as well, so it will be very helpful. I will be waiting for that before I can continue with this PR :)